README.md

```diff
@@ -21,7 +21,71 @@ This node is used by Rootstock PowPeg signatories to interact with the Bridge co
+   5. Unzip it like this: `sudo tar -zxvf bitcoind18.tar.gz`
+   6. Run the `ls` command to see the actual name of the unzipped directory. It should be like `bitcoin-0.18.1`
+   7. Add the path `export PATH="$PATH:/Library/Bitcoin/bitcoin-0.18.1/bin"` to the `/.zshrc` file
+      1. Open the `.zshrc` file with nano like this: `nano ~/.zshrc` and add the following line in it and save it: `export PATH="$PATH:/Library/Bitcoin/bitcoin-0.18.1/bin"`
+   8. Run `source ~/.zshrc` command, close and reopen the terminal.
+   9. Running an instance of bitcoind
+      1. To run it in regtest mode, run this command: `bitcoind -deprecatedrpc=generate -addresstype=legacy -regtest -printtoconsole -server -rpcuser=rsk -rpcpassword=rsk -rpcport=18332 -txindex -datadir=/Library/Bitcoin/data`
+      2. You can also put this command into an `bitcoin-regtest.sh` file, make it executable with `sudo chmod +x bitcoin-regtest.sh` and run it like `./bitcoin-regtest.sh`
+      3. To run it in regular mode, simply run the command: `bitcoind`
+   10. Generate some blocks
+       1. To generate, for example, 200 regtest bitcoin blocks, run: `./btc-regtest.sh generate 200`
+3. Create a `powpeg-project` folder anywhere you like
+   1. For example: `mkdir /Library/powpeg-project`
+4. Setup the rskj project
+   1. `cd` to the `powpeg-project` directory: `cd /Library/powpeg-project`
+   2. Clone it from here: `https://github.com/rsksmart/rskj`
+      1. `git clone https://github.com/rsksmart/rskj.git`
+      2. `cd` to the cloned `rskj` directory: `cd rskj`
+   3. Run the `configure.sh`
+      1. You will probably have to make it executable first with: `sudo chmod +x configure.sh`
+      2. And run it from a terminal like this: `./configure.sh`
+5. Setup this project (powpeg-node: `https://github.com/rsksmart/powpeg-node`)
+   1. `cd` to the `powpeg-project` directory: `cd /Library/powpeg-project`
+   2. `git clone https://github.com/rsksmart/powpeg-node.git`
+   3. `cd` to the cloned `powpeg-node` directory: `cd powpeg-node`
+   4. Run the `configure.sh` file present in the root directory
+      1. You will probably have to make it executable with: `sudo chmod +x configure.sh`
+      2. And run it from a terminal like this: `./configure.sh`
+   5. Make a copy of the `development-settings.gradle.sample` file and rename it to `DONT-COMMIT-settings.gradle`
+      1. Remove the line `# Sample configuration to build rskj from the directory /home/user/another/dir/rskj`
+      2. Remove the line `# Rename it to DONT-COMMIT-settings.gradle for use in your local environment`
+      3. Replace the `'/home/user/another/dir/rskj/'` with the relative or absolute path where the `rskj` project is, for example: `/Library/powpeg-project/rskj`
+   6. Create a `fed.conf` file and set it up
+      1. Check the config file sample in `src/main/resources/config/fed-sample.conf`, copy it, rename it to `fed.conf` and update it as you need.
+   7. Optionally create a `logback.xml` file for the logging
```

Comment:
I feel like this should not be optional and instead a required. When I tried running without logback I would get nothing to stdout, and once I set it up as explained here, I started to get log output displayed.